trashcli/put.py

```diff
@@ -250,71 +254,50 @@ def volume_of_parent(self, file):
 
 
 def get_option_parser(program_name, stdout, stderr):
-    from optparse import OptionParser
-
-    parser = OptionParser(prog=program_name,
-                          usage="%prog [OPTION]... FILE...",
-                          description="Put files in trash",
-                          version="%%prog %s" % version,
-                          formatter=NoWrapFormatter(),
-                          epilog="""\
-To remove a file whose name starts with a '-', for example '-foo',
-use one of these commands:
-
-    trash -- -foo
-
-    trash ./-foo
-
-Report bugs to https://github.com/andreafrancia/trash-cli/issues""")
-
-    parser.add_option("-d", "--directory",
-                      action="store_true",
-                      help="ignored (for GNU rm compatibility)")
-    parser.add_option("-f", "--force",
-                      action="store_const",
-                      dest="mode",
-                      const=mode_force,
-                      help="silently ignore nonexistent files")
-    parser.add_option("-i", "--interactive",
-                      action="store_const",
-                      dest="mode",
-                      const=mode_interactive,
-                      help="prompt before every removal")
-    parser.add_option("-r", "-R", "--recursive",
-                      action="store_true",
-                      help="ignored (for GNU rm compatibility)")
-    parser.add_option("--trash-dir",
-                      type='string',
-                      action="store", dest='trashdir',
-                      help='use TRASHDIR as trash folder')
-    parser.add_option("-v",
-                      "--verbose",
-                      default=0,
-                      action="count",
-                      dest="verbose",
-                      help="explain what is being done")
-    parser.add_option('--force-volume',
-                      default=None,
-                      action="store",
-                      dest="forced_volume",
-                      help=SUPPRESS_HELP)
-    original_print_help = parser.print_help
-
-    def patched_print_help():
-        original_print_help(stdout)
-
-    def patched_error(msg):
-        parser.print_usage(stderr)
-        parser.exit(2, "%s: error: %s\n" % (program_name, msg))
-
-    def patched_exit(status=0, msg=None):
-        if msg: stderr.write(msg)
-        import sys
-        sys.exit(status)
-
-    parser.print_help = patched_print_help
-    parser.error = patched_error
-    parser.exit = patched_exit
+    from argparse import ArgumentParser, RawDescriptionHelpFormatter, SUPPRESS
+
+    parser = ArgumentParser(prog=program_name,
+                            usage="%(prog)s [OPTION]... FILE...",
+                            formatter_class=RawDescriptionHelpFormatter,
+                            description="Put files in trash",
+                            epilog=__doc__)
+
+    parser.add_argument("-V", "--version",
```

Comment:
Please remove "-V" this is a change in the behaviour. This pull request should be just about refactorings.

Reply:
Fixed.

Reply:
Good